pkgs/by-name/gi/git-fixup/package.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, stdenv, fetchFromGitHub, makeWrapper, git, coreutils, gnused, gnugrep }:
+
+stdenv.mkDerivation (finalAttrs: {
```

Comment:
```suggestion
stdenvNoCC.mkDerivation (finalAttrs: {
```

Reply:
Thanks for the hint, I've applied it and rebased the branch.